Cargo.lock

```diff
@@ -9,7 +9,7 @@ dependencies = [
  "env_logger 0.3.2 (registry+https://github.com/rust-lang/crates.io-index)",
```

Comment:
Please ignore changes in ```Cargo.lock``` and ```Cargo.toml``` - they are here to include ```users``` patch that allows passing ```auth``` token as query string parameter - I'll get rid of these changes for final PR.